packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -104,14 +105,16 @@ export const AuthContextProvider = ({
   getRedirectUri,
   visit,
 }: AuthContextProviderProps): ReactElement => {
+  const [isCompletionOpen, setIsCompletionOpen] = useState(false);
```

Comment:
This is a bit strange naming I think.
Perhaps we can make it isIncompleteRegistration ?

Reply:
Totally agree with that. Let me refactor it first! 🚢